media-gfx/sigal/sigal-2.0.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Simple static web gallery generator"
+HOMEPAGE="https://sigal.saimon.org/"
+SRC_URI="https://github.com/saimn/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="brotli crypt rss s3 test zopfli"
+RESTRICT="!test? ( test )"
+
+CDEPEND="dev-python/blinker[${PYTHON_USEDEP}]
```

Comment:
Use `COMMON_DEPEND` instead.

Although this ebuild has no reason to use `CDEPEND` (or `COMMON_DEPEND`).